### PR TITLE
67 add ground truth for cosmos region annotation tests

### LINKED ISSUE #67: Add ground truth for COSMOS region annotation tests
Ian has manually annotated the [SIDARTHE paper](https://www.nature.com/articles/s41591-020-0883-7.pdf) and a markdown document for the [BUCKY model](https://github.com/UW-COSMOS/Cosmos/blob/master/cosmos_service/test/resources/pdfs/bucky.pdf) with expected bounding boxes and region classifications. 

The annotations are currently stored as a series of per-pdf, per-page XML documents in the XDD COSMOS repo:
https://github.com/UW-COSMOS/Cosmos/tree/master/cosmos_service/test/resources/annotated/sidarthe

Tests against these manual annotations for 3 region classifications (equations, tables, and figures) are implemented in the XDD COSMOS repo:
https://github.com/UW-COSMOS/Cosmos/blob/master/cosmos_service/test/src/annotations/test_sidarthe.py

These tests check for at least 90% agreement between COSMOS and the manual annotations on each page of each given document, and fail if any page is at under 90% agreement. This may or may not be a suitable measurement for the knowledge middleware dashboard. 

The `test_pdf_to_cosmos` test should be updated to:
- Fetch the manual region annotations for the document being processed
- Compare the bounding boxes generated by COSMOS to the manually annotated documents
- Determine the degree to which COSMOS output agrees with the manual annotations based on some metric
- Store results so they can be used in the integration dashboard

### PR DESCRIPTION
**Pull request by Sorrento110**
# Description

* Added cosmos accuracy measures into the test pipeline. 

Resolves #67

### CODE REVIEW

**Review by brandomr — CHANGES_REQUESTED** [review]

File: tests/scenarios/sidarthe/cosmos_ground_truth/stub.yaml
[the remark this answers is not in the document]
Comment (Sorrento110) [reply tests/scenarios/sidarthe/cosmos_ground_truth/stub.yaml:1]: It was in the `reporting/scenarios/SIDARTHE` directory for the pipeline, I have moved it into both locations.

File: reporting/tests/report.py [hunk 1]
```diff
@@ -140,9 +140,43 @@ def do_task(url, task, kwargs={}):
...
+                            "document_overlap_percent"
+                        ],
+                        "document_expected_count": evaluation[
+                            "document_expected_count"
+                        ],
+                        "document_cosmos_count": evaluation["document_cosmos_count"],
+                    }
+                    result["accuracy"] = evaluation_stats
+                else:
+                    result["accuracy"] = {"status_code": eval.status_code}
```
Comment (brandomr) [inline reporting/tests/report.py:177]: My worry is that the `status_code` might show up in the dashboard and cause some confusion. Or is it handled somewhere else? Maybe we should put `Failed` or something like that instead?

File: tests/test_e2e.py [hunk 2]
```diff
@@ -166,6 +166,24 @@ def test_pdf_to_cosmos(
...
+    #### POSTAMBLE ####
+    # STUB
+    # if not settings.MOCK_TA1:
+    #     with open(f"{context_dir}/cosmos_ground_truth/ground_truth.yaml") as f:
+    #         ground_truth = yaml.load(f)
+
+    #         # Post ground truth to Cosmos Eval endpoint
+    #         response = requests.post(
+    #             f"{settings.COSMOS_URL}/eval", json=json.loads(json.dumps(ground_truth))
+    #         )
```
Comment (brandomr) [inline tests/test_e2e.py:178]: Can we just remove this block since it won't be used?

**Review by brandomr — COMMENTED** [review]
Looks good, just a couple comments

**Review by brandomr — APPROVED** [review]

**Pull request merged**